Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -820,11 +818,16 @@ private void HandleOrderEvent(OrderEvent fill)
             {
                 Log.Debug("BrokerageTransactionHandler.HandleOrderEvent(): " + fill);
                 Interlocked.Exchange(ref _lastFillTimeTicks, DateTime.Now.Ticks);
+
+                // set the price currency on the fill and on the order
+                var security = _algorithm.Securities[fill.Symbol];
+                fill.FillPriceCurrency = security.SymbolProperties.QuoteCurrency;
+                order.PriceCurrency = fill.FillPriceCurrency;
```

Comment:
This makes sense here since it's in a single place but, a piece of me thinks that the brokerages should be populating this field when they produce the fill.